packages/framework/src/Framework/Features/Publications/Models/PublicationFieldType.php

```diff
@@ -64,25 +65,26 @@ public function toArray(): array
         ];
     }
 
+    /** @see \Hyde\Framework\Testing\Unit\PublicationFieldTypeValidationRulesTest */
     public function getValidationRules(bool $reload = true): Collection
     {
         $defaultRules = Collection::create(PublicationFieldTypes::values());
         $fieldRules = Collection::create($defaultRules->get($this->type->value));
 
-        $doBetween = true;
+        $useRange = true;
         // The trim command used to process the min/max input results in a string, so
         // we need to test both int and string values to determine required status.
         if (($this->min && ! $this->max) || ($this->min == '0' && $this->max == '0')) {
             $fieldRules->forget($fieldRules->search('required'));
-            $doBetween = false;
+            $useRange = false;
         }
 
         switch ($this->type->value) {
             case 'array':
-                $fieldRules->add('array');
+                $fieldRules->add('array'); // FIXME do we do range validation too?
```

Comment:
Should we do range validation on arrays too?

Reply:
Well, this has the potential to open a can of works. Because what do we mean? The # if items in the array? The lengths of the individual items in the array? Both? I'd rather just let the user enter whatever, because we could just be adding unnecessary complexity.

Reply:
I was thinking it would limit the number of entries in the array. But you're right, it likely adds more complexity than value.